inference-engine/tests/functional/plugin/conformance/subgraphs_dumper/src/main.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <fstream>
+#include <regex>
+
+#include "inference_engine.hpp"
+
+#include "common_test_utils/file_utils.hpp"
+
+#include "ops_cache.hpp"
+#include "gflag_config.hpp"
+
+// TODO: Poor exceptions handling
+int main(int argc, char *argv[]) {
+    uint8_t ret_code = 0;
+
+
+    gflags::ParseCommandLineNonHelpFlags(&argc, &argv, true);
+    if (FLAGS_h) {
+        showUsage();
+        return 0;
+    }
+    auto ie = InferenceEngine::Core();
+
+    auto cache = SubgraphsDumper::OPCache::make_cache();
+
+    if (!CommonTestUtils::directoryExists(FLAGS_input_folders)) {
```

Comment:
`FLAGS_input_folders` -> `std::vector<std::string>`. I mean it is possible to have some number of `input_folders`